client/components/common/BlockForm.vue

```diff
@@ -521,7 +522,7 @@ export default {
         }
         
         if (this.refreshRecipes) {
-          this.$store.commit('refreshRecipes');
+          await this.$store.commit('fetchRecipes', this.showLoading ? this.showLoading : false);
```

Comment:
Where is this.showLoading coming from in block form?

Reply:
updated, it's legacy from prev idea i had in mind